app/assets/stylesheets/custom.scss

```diff
@@ -0,0 +1,41 @@
+/* universal */
+
+body {
+    padding-top: 60px;
```

Comment:
please tab 2 space

Reply:
Is this means for the function which I tab 4 space just like the second comment? Sorry, do not really understand the sentence.

Reply:
yes, you should 1 tab = 2 spaces.

Current, you use 1 tab = 4 spaces. plz, change it :D